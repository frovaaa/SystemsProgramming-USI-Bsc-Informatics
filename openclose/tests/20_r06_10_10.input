90027
26pq59
35894
52180
8b225d
41391
7b621d
76056
28354
58094
