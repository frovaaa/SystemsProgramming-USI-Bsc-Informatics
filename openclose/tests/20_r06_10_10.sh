#!/bin/bash
exec "$1" "bdpq" < tests/20_r06_10_10.input
