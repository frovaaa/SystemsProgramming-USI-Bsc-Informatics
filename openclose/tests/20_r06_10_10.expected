90027
2659
35894
52180
8b225d
41391
7b621d
76056
28354
58094
